packages/tracking/composer.json

```diff
@@ -4,15 +4,17 @@
 	"type": "library",
 	"license": "GPL-2.0-or-later",
 	"require": {
-		"automattic/jetpack-options": "@dev"
+		"automattic/jetpack-options": "@dev",
+		"automattic/jetpack-connection": "@dev"
 	},
 	"require-dev": {
 		"phpunit/phpunit": "^5.7 || ^6.5 || ^7.5",
 		"php-mock/php-mock": "^2.1"
 	},
 	"autoload": {
 		"psr-4": {
-			"Automattic\\Jetpack\\": "src/"
+			"Automattic\\Jetpack\\": "src/",
```

Comment:
Hm, different namespaces within a small package....shouldn't we be using the same namespace? I think we could use the `Automattic\Jetpack\Tracking` namespace for all of these, just like we do for the connection package.